test/externalTests/common.sh

```diff
@@ -207,7 +207,7 @@ function run_test
     for optimize in "${optimizer_settings[@]}"
     do
         clean
-        force_solc_settings "$CONFIG" "$optimize" "petersburg"
+        force_solc_settings "$CONFIG" "$optimize" "istanbul"
```

Comment:
This will probably break some tests, but what could one do :wink:

Reply:
I'm unsure about the changes in the test scripts.